examples/case_studies/item_response_nba.ipynb

```diff
@@ -0,0 +1,1142 @@
+{
```

Comment:
<p>here it would be better if you could use <code>os.path.join</code>  to read the file as mentioned in the style guide here: https://github.com/pymc-devs/pymc3/wiki/PyMC3-Jupyter-Notebook-Style-Guide  as using <code>/</code>  breaks on Windows.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/181/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>